lib/ffprobe.ex

```diff
@@ -52,6 +53,26 @@ defmodule FFprobe do
     |> Enum.into(%{})
   end
 
+  @spec format_json(binary) :: format_json
+  def format_json(file_path) do
+    cmd_args = ["-v", "quiet", "-print_format", "json", "-show_format", file_path]
+    {result, 0} = System.cmd ffprobe_path(), cmd_args, stderr_to_stdout: true
+
+    result
+    |> Poison.decode!()
+    |> Map.get("format", %{})
+  end
+
+  @spec streams_json(binary) :: format_json
```

Comment:
Please rename `streams_json` -> `streams` and make the typespec clear that it returns a list.